ivadomed/evaluation.py

```diff
@@ -444,21 +444,24 @@ def run_eval(self):
             dict, ndarray: dictionary containing evaluation results, data with each object painted a different color
         """
         dct = {}
-
+        data_gt = self.data_gt.copy()
```

Comment:
Have you checked if still okay with one-class segmentation tasks? Otherwise happy to have a look at it!